digits/model/forms.py

```diff
@@ -175,14 +185,39 @@ def validate_lr_multistep_values(form, field):
                 ],
             )
 
-    custom_network = wtforms.TextAreaField('Custom Network',
+    custom_network = dforms.TextAreaField('Custom Network',
             validators = [
                 validate_required_iff(method='custom'),
                 validate_NetParameter,
-                ]
+                ],
+            explanation =
+                        '    <h1>Specifying a custom network</h1>\n'
+                        '    <p>\n'
+                        '    You should enter a train_val network here. Some cleanup is done for you automatically:\n'
+                        '    <ul>\n'
+                        '        <li>\n'
+                        '        The <i>num_output</i> for each <b>InnerProduct</b> layer which is a network output gets set to the number of labels in the chosen dataset.\n'
+                        '        </li>\n'
+                        '        <li>\n'
+                        '        Any <b>Accuracy</b> layers with <i>top_k</i> &gt;= num_labels is removed automatically.\n'
+                        '        </li>\n'
+                        '    </ul>\n'
+                        '    </p>\n'
+                        '    The deploy network will be created for you automatically. Once caffe implements their "all-in-one" nets feature, you will be able to specify which layers to include in the deploy file. Until then, these things are done automatically:\n'
+                        '    <ul>\n'
+                        '        <li>\n'
+                        '        The last <b>SoftmaxWithLoss</b> layer is replaced with a <b>Softmax</b> layer.\n'
+                        '        </li>\n'
+                        '        <li>\n'
+                        '        All other loss layers and accuracy layers are removed.\n'
+                        '        </li>\n'
+                        '    </ul>\n'
+                        '    </p>\n'
```

Comment:
Can we put this in a template and render it to a string like this?
https://github.com/NVIDIA/DIGITS/blob/v2.0.0-rc3/digits/task.py#L99-L104